src/euphorie/client/docx/compiler.py

```diff
@@ -502,7 +501,7 @@ def compile(self, data):
         self.set_consultation_box()
 
 
-class DocxCompilerItaly(DocxCompiler):
+class DocxCompilerItalyOriginal(DocxCompiler):
```

Comment:
Can this be removed? I do not see it referenced anywhere else in this PR.

Reply:
Thanks for the hint, I fixed it, now it's used again.